<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consume_Vip_Record</class>
 <widget class="QMainWindow" name="Consume_Vip_Record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2001</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找该月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择要查找的日期</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找该年</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy</string>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找该天</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="qTableWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>731</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="qTableWidget2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>731</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
